keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -881,6 +887,74 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Instrumentation
+
+The kubelet will record the following metrics:
+
+#### `kubelet_container_resize_attempts_total`
```

Comment:
Attempts makes me think we record this each time a deferred resize is reevaluated, but I don't think that's the intention. I'm having a hard time coming up with a better option though...
- `kubelet_container_resize_requests_total` - I think this was the original, but it's not HTTP requests
- `kubelet_container_resizes_total` - Sounds like it's counting completed resizes
- `kubelet_container_resize_updates_total` - Maybe this one?